app/views/layouts/default.html.erb

```diff
@@ -136,9 +136,6 @@
     <a href="#content" class="show-with-keyboard-focus-only skip-to-link">
       <%= _('Back to content') %>
     </a>
-    <script>
-      $(document).foundation();
```

Comment:
Where do we load foundation now? Is this removal related to the removal of foundation in https://github.com/mysociety/alaveteli/pull/3451/commits/e0eb7e34760b1c4fa46635f1fd83f8b4f2b5264f#r77012948?